src/models/character.ts

```diff
@@ -67,6 +67,7 @@ export class Character {
 
   sack: Item[] = [];
   belt: Item[] = [];
+  locker: Item[] = [];
```

Comment:
This should be locker: any = {} where the key in the object is the identifier for the locker, mapped to an array.